cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -12,7 +12,9 @@ describe.skip('FluxQueryBuilder', () => {
           }).then(() => {
             const writeData = []
             for (let i = 0; i < 30; i++) {
-              writeData.push(`ndbc,air_temp_degc=${i}_degrees station_id=${i}`)
+              writeData.push(
+                `ndbc,air_temp_degc=70_degrees station_id_${i}=${i}`
```

Comment:
I updated the line protocol here to have *one* tag value and *30* fields

Reply:
how come?

Reply:
@drdelambre the old line protocol was generating *30* tag values and *one* field, so we were testing "Load more" button under tag values. Cypress was super wired when it 'click()' on "Load more" button, the tag value list got un-expanded, so "Load more" button DOM disappeared and the test failed at this point.

Then I thought to test the "Load more" button that is under fields to walk around the wired behavior. That's why I updated the line protocol